app/models/asset.rb

```diff
@@ -32,6 +32,8 @@ class Asset
 
   field :access_limited, type: Array, default: []
 
+  field :parent_document_url, type: String
```

Comment:
Do you think it is worth adding any validation to ensure that the value stored is a valid URL?

Reply:
Yeah, I reckon it is. I did consider initially but discounted it for reasons I no longer remember. I'll add it to this PR.

Reply:
I also considered doing this for `Asset#redirect_url`, but discounted it, because I thought it would be complicated to know what hosts were allowed in the various environments + draft/public stacks. Are you imagining just checking it's a valid URL or that it's a valid GOV.UK URL?

Reply:
Probably just that it's a valid url - so that any header added is valid against https://tools.ietf.org/html/rfc5988#section-5.

Reply:
Do you think we should add some validation for `Asset#redirect_url` and, if so, what?

Reply:
I remember now - it's because it's not particularly easy to add such validation :-)

@chrislo - What do you think of the validation I've added in https://github.com/alphagov/asset-manager/pull/502/commits/f51bb68965e01280aec9ec72b14068341000e57a?

Reply:
Sorry, @floehopper. I've only just seen your comments (even though they were posted before mine). What do you think about the validation I've added? Does it seem reasonable?

Reply:
It seems OK to me. Did you consider using a validator class? There might be some inspiration in https://github.com/perfectline/validates_url or it might even be worth using it...?

Reply:
I thought a validator class was overkill for validating a single attribute, but can see that it might be worth introducing if we do need to validate multiple URLs.

I've had a quick look at the validates_url Gem. It seems reasonable but I don't feel particularly strongly about whether we should use it.